src/storage/src/store.rs

```diff
@@ -377,14 +381,35 @@ pub struct NewLocalOptions {
     pub table_option: TableOption,
 }
 
+#[derive(Clone)]
+pub struct NewLocalOptions {
+    pub table_options: NewLocalTableOptions,
```

Comment:
> Any specific reason to introduce a new struct `NewLocalTableOptions` instead of including the newly added field to the flattened `NewLocalOptions`?

`LocalHummockStorage` and `NewLocalTableOptions` only needs `NewLocalTableOptions` to initialize. If we pass a `NewLocalOptions` to them on initialization, it will violates the rule of encapsulation.

Reply:
I think there is no break of encapsulation since it does no harm to pass the vnode distribution to local hummock storage. Introducing a new concept is confusing and increases complexity.

Reply:
> I think there is no break of encapsulation since it does no harm to pass the vnode distribution to local hummock storage. Introducing a new concept is confusing and increases complexity.

If including the newly added field to the flattened `NewLocalOption`, in `new_local_inner`, we need to move out `vnodes` field in the `NewLocalOption` and then pass the clone of `NewLocalOption` to `LocalHummockStorage::new`.